pkg/mocks/Makefile

```diff
@@ -0,0 +1,6 @@
+.PHONY: mocks
```

Comment:
Do we need this here and in the top level Makefile? Seems simpler to maintain it in a single place

Reply:
Good call. Will move to top level